src/Tkmm/Wizard/NxSetupWizard.cs

```diff
@@ -0,0 +1,78 @@
+#if SWITCH
+using Avalonia.Controls.Presenters;
+using Tkmm.Core;
+using Tkmm.Models.MenuModels;
+using Tkmm.Wizard.Pages;
+using TkSharp.Extensions.LibHac.Util;
+
+namespace Tkmm.Wizard
+{
+    public sealed class NxSetupWizard(ContentPresenter presenter) : SetupWizard(presenter)
+    {
+        public override async ValueTask Start()
+        {
+            TkConfig.Shared.SdCardRootPath = "/flash";
+            TkConfig.Shared.KeysFolderPath = "/flash/switch";
+
+        FirstPage:
+            await FirstPage();
+            
+            if (!TkKeyUtils.TryGetKeys(TkConfig.Shared.SdCardRootPath, out var keys))
+            {
+                bool proceed = await NextPage()
+                    .WithTitle(TkLocale.SetupWizard_MissingKeys_Title)
+                    .WithContent(TkLocale.SetupWizard_MissingKeys_Content)
+                    .WithActionContent(TkLocale.Menu_NxReboot)
+                    .Show();
+                if (!proceed)
+                {
```

Comment:
Fix code formatting.

```suggestion
                    .Show();
                
                if (!proceed) {
```